code/modules/clothing/neck/skillcapes/skillcape_datums.dm

```diff
@@ -108,6 +108,16 @@
     job = "Detective"
     path = /obj/item/clothing/neck/skillcape/trimmed/detective
 
+/datum/skillcape/physician
+    name = "cape of the brig physician"
+    job = "Detective"
+    path = /obj/item/clothing/neck/skillcape/physician
+
+/datum/skillcape/trimmed/physician
+    name = "cape of the grand brig physician"
+    job = "Detective"
```

Comment:
neither does this

Reply:
this was all copied from the dets skill cape, and I must have forgotten to change it